docs/src/main.css

```diff
@@ -30,7 +30,7 @@ code {
 }
 
 .grid-container {
-  background-color: #ddd;
+  background-color: white;
```

Comment:
First, thanks for accepting this trial of Hercules! Now, on the comments!

Dunno why this is like this — I inherited it. 
Colors in the main.css file in the docs aren't run through a sass preprocessor, so we can't use the sass variables. Otherwise yeah, I try to use vars everywhere possible. 

THAT SAID — this is only the stylesheet for the PL, so it's pretty sloppy and low priority!